docs/registries.conf.5.md

```diff
@@ -0,0 +1,28 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is TOML.
+
+# FORMAT
+The [TOML_format][toml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`. You can list multiple registries using
+as a comma separated list.
+
+The following example configuration defines two registries and one insecure registry.
+
+```
+[registries.search]
+registries = ["registry1.com", "registry2.com"]
+
+[registries.insecure]
+registries = ["registry3.com"]
```

Comment:
The above does not really say much about what the values mean; that is, instead, contained in the comments inside the file. That’s a bit unusual.

If the substantive documentation remains in the config file comments, this man page should at least mention that, so that users don’t go looking elsewhere.

Though I’d be happier if the core of the documentation (except for the block_registries block, I guess) were in the man page, and the config file pointed to the man page, instead of the other way around: The comments in the config file can easily get lost, e.g. when a tool overwrites the file with a generated version. The man page, OTOH will end up on various man page servers / search systems.

Reply:
This is an excellent comment and on point.  I've taken the search and insecure registry descriptions with comments from above fixed and added them to the man page.  I've put a very abbreviated summary in the configuration file instead and will mention that further description is available in the man pages.  I left the blocked description out of the man page as it is being deprecated but left the full description in the conf file.